### ISSUE TITLE
gulp build redirect failure

### ISSUE THREAD

**Issue by Zwoleo** [mgechev/angular2-seed#1121]
When I am running gulp build.prod everything works fine.
Since I started to using gradle, standard task for gradle is gulp build, so I decided to create task:

```
gulp.task('build', (done: any) =>
  runSequence('build.prod',
              done));
```

When I am trying to run gulp build it fails on:

```
[15:13:50] Starting 'build.js.prod'...
[15:13:51] Plumber found unhandled error:
 Error in plugin 'gulp-inline-ng2-template'
Message:
    ENOENT: no such file or directory, open xxx
Details:
    errno: -4058
    code: ENOENT
    syscall: open
    path: xxx
    fileName: xxx
```

I can't understand that, it should only redirect it to another task.

**Comment by TheDonDope** [comment]
Hi @Zwoleo,

that is not quite the way it works. The seed build system works like this:
There are the "main" gulp tasks, for example `build.dev`, `build.prod`, `build.test` etc. Those tasks wrap a sequence of tasks that will then be executed in the order they appear.

For example for `build.prod`:

```TypeScript
// --------------
// Build prod.
gulp.task('build.prod', (done: any) =>
  runSequence('clean.prod',
              'tslint',
              'css-lint',
              'build.assets.prod',
              'build.html_css',
              'copy.js.prod',
              'build.js.prod',
              'build.bundles',
              'build.bundles.app',
              'build.index.prod',
              done));
```

Those individual tasks then get mapped to actual file names (hint: take a look at the `tools/tasks/seed` directory, and you will find a corresponding `.ts` file for those tasks). 

However, since the "main" tasks themselves are mere "wrapper" tasks, they do not map to an individual typescript file. Hence you get the `ENOENT: no such file or directory` error.

Long story short, to get the behaviour you want, you will need to copy the task list from the `gulp.prod` task and enter it as the parameter for your `runSequence` call.

Greetings

Dope

**Comment by Zwoleo** [comment]
Hello, 
funny thing is that I tried like u said and it still doesnt work.
Important in this case maybe could be that error:

```
 Error in plugin 'gulp-inline-ng2-template'
Message:
    ENOENT: no such file or directory, open 'C:\PROJEKTY\test\dist\tmp\app\xxx.css'
```

dist/tmp includes only .ts files. There is no css/jss files. Because error fails on css and js files.
